src/api/spec/controllers/webui/image_templates/interconnects_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+require './spec/support/shared_examples/controllers/image_templates.rb'
```

Comment:
usually all the stuff in spec/support is required by default by the spec_helper, for some reason we're not doing that but I think you can add this line to spec/spec_helper.rb

```ruby
Dir["./spec/support/shared_examples/*.rb"].each { |file| require file }
```